ccdproc/combiner.py

```diff
@@ -150,15 +149,12 @@ def scaling(self):
 
         Parameters
         ----------
-
-        scale : function or array-like or None, optional
+        scale : function or `numpy.ndarray`-like or None, optional
             Images are multiplied by scaling prior to combining
             them. Scaling may be either a function, which will be applied to
             each image to determine the scaling factor, or a list or array
             whose length is the number of images in the `~ccdproc.Combiner`.
-            Default is `None`.
```

Comment:
Is there a default for properties? The default is only valid for `__init__` but the setter doesn't have a default as far as I can see.